### ISSUE TITLE
Fix RealmInterprocessTest

### ISSUE THREAD

**Issue by cmelchior** [realm/realm-java#6452]
After #6439 is merged the RealmInterprocessTest is broken. We need to re-enable it.

**Comment by marcus-hiles** [comment]
Yeah, agreed @cmelchior. Hope you've been able to implement a fix for it? :-)
